stitching_base/Dockerfile

```diff
@@ -1,27 +1,40 @@
-FROM bepro/tensorflow_cc:ubuntu-cuda-2.5.0
+FROM floopcz/tensorflow_cc:ubuntu-cuda-2.6.0
```

Comment:
The recent version includes the 8.6 capability which is for RTX 3080 as the default value.
https://github.com/FloopCZ/tensorflow_cc/commit/bcf01eb552ce79a3dfc83a417a6f6858cc811f14
https://hub.docker.com/layers/floopcz/tensorflow_cc/ubuntu-cuda-2.6.0/images/sha256-8d61195134bed90ed1568c0a2882b6f8f8e0a9fae4546586c2453ff545a3c865?context=explore

So we don't need to use our own docker image.